GoedeDoelenHelpen/ClientApp/src/app/user/dashboard/header/header.component.html

```diff
@@ -3,18 +3,20 @@
     <p class="welcome-message">Goedemorgen Koen!</p>
     <div class="page-select">
       <mat-form-field>
-          <mat-select placeholder="Pagina">
-            <mat-option *ngFor="let page of pages" [value]="page.id">
-              {{ page.title }}
-            </mat-option>
-          </mat-select>
-        </mat-form-field>
-        <div class="page-buttons">
-          <button mat-button class="dashboard-button">
-            <img src="{{ images }}/view.svg" alt="Bekijk eventpagina"></button>
-          <button mat-button class="dashboard-button">
-            <img src="{{ images }}/edit.svg" alt="Bewerk eventpagina"></button>
-        </div>
+        <mat-select placeholder="Pagina">
+          <mat-option *ngFor="let page of pages" [value]="page.id">
+            {{ page.title }}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+      <div class="page-buttons">
+        <button mat-button class="dashboard-button">
+          <img src="{{ images }}/view.svg" alt="Bekijk eventpagina"></button>
+        <button mat-button class="dashboard-button">
+          <img src="{{ images }}/edit.svg" alt="Bewerk eventpagina"></button>
+        <button mat-button class="fbauth-button" (click)="fbAuth()">
+          <img src="{{ images }}/edit.svg" alt="FB"></button>
```

Comment:
edit.svg is niet representatief voor wat de knop hoort te doen, vraag bij Kristy om een goede svg icon voor facebook en verwerk deze. alt="FB" is niet duidelijk genoeg voor wat de knop doet.